dmlproc/dmlprocessor.cpp

```diff
@@ -1845,7 +1845,7 @@ void DMLProcessor::operator()()
 
 void RollbackTransactionProcessor::processBulkRollback(BRM::TableLockInfo lockInfo, BRM::DBRM* dbrm,
                                                        uint64_t uniqueId,
-                                                       OamCache::dbRootPMMap_t& dbRootPMMap,
+                                                       OamCache* oamcache,
```

Comment:
nit: `const OamCache*`?

Reply:
Why?

We can reload, if we implement on-the-fly addition of nodes the OAMCache (or similar object) will not be const.

Reply:
Do we really want to reload in the middle of an operation?